docs/src/api.md

```diff
@@ -8,4 +8,6 @@ Blast.get_clencurt_grid
 Blast.w_ell_tullio
 Blast.plan_fft
 Blast.fast_chebcoefs
+Blast.SimpsonWeightArray
```

Comment:
Change the name of the `SimpsonWeightArray` method., as it is not compliant to style guidelines.